testsuite/tests/typing-implicit-source-positions/expressions.ml

```diff
@@ -27,6 +27,31 @@ let _ = f ~call_pos:[%src_pos] () ;;
 {pos_fname = ""; pos_lnum = 2; pos_bol = 438; pos_cnum = 458}
 |}]
 
+(* passing an argment explicitly as call_pos *)
```

Comment:
I would want a test showing that `[%call_pos]` is not accepted as a type in other positions